homeassistant/core.py

```diff
@@ -648,15 +648,23 @@ def is_state(self, entity_id, state):
         Async friendly.
         """
         state_obj = self.get(entity_id)
-        return state_obj and state_obj.state == state
```

Comment:
Ah yes, I got bitten by that the other day too. Apparently `None and True` will return `None` instead of `False`. Anyway, instead of writing it out, can you change it to:

```python
return state_obj is not None and state_obj.state == state
```